### ISSUE TITLE
Update Secret Handling to _FILE convention

### ISSUE THREAD

**Issue by mjclemente** [Ortus-Solutions/docker-commandbox#44]
This isn't an issue, but more of a feature request (that would be a breaking change, but since it seems there's a major release coming, it might make sense to consider now).

It seems a lot of Docker images have adopted the `_FILE` syntax for automatically expanding secrets. That is, you define secrets in compose like this:

```
environment:
      - MONGO_USERNAME_FILE=/run/secrets/mongo_root
```
And based on that file suffix, the secret is expanded and the variable is set.

You can see examples of the expansion here, in [Bret Fisher's node example](https://github.com/BretFisher/node-docker-good-defaults/blob/master/docker-entrypoint.sh) and in the [Postgres image](https://github.com/docker-library/postgres/blob/04f28840134bdaf72e4cd6e4a5300e9ceb5ea013/11/docker-entrypoint.sh). I've seen this in a number of other images as well, including MySQL, Wordpress, and RabbitMQ.

**Comment by jclausen** [comment]
We could definitely implement that, though we're already parsing the entire secrets directory at present: https://github.com/Ortus-Solutions/docker-commandbox/blob/development/build/util/env-secrets-expand.sh

I'm wondering if would make sense to add a convention that parallels or remove the old expansion - which would be a breaking change

**Comment by mjclemente** [comment]
I had posted this with the though that, if v3.0.0 is going to be breaking already, adjusting the secrets syntax seems like a fairly minor adjustment. That said, backwards compatibility is always appreciated and seems to be a core tenant of CFML related development, so if there's no need to make it a breaking change, maybe supporting both for a period of time would be better, for the sake of current developers. Perhaps with the goal of migrating toward the more widely adopted approach at some future date/release.

We're currently using the existing <<>> approach for secrets, but would make the change, but I could see that being a pain for people handling lots of deployments. I guess ultimately it comes down to your plan for and usage of this image. 

Thanks for all the work you do on it. Much appreciated.

**Comment by jclausen** [comment]
This feature will be released in  v3.0.0 ( https://ortussolutions.atlassian.net/browse/OC-10 )

**Comment by mjclemente** [comment]
Thanks!
